src/styles/blocks/card.scss

```diff
@@ -0,0 +1,84 @@
+@import "../utils/colors";
+
+.card {
+  width: 200px;
+  padding: 0 16px;
+  flex-shrink: 0;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background: $white;
+
+  &__product-img {
+    margin: 32px auto 0;
+    width: 160px;
+    height: 134px;
+    display: flex;
+  }
+
+  &__product-name {
+    margin: 40px 0 4px;
+    width: 100%;
+    height: 36px;
```

Comment:
```suggestion

```
The height of the block should depend on the content, consider using margins or paddings to make indents between blocks or line-height to increase the height of the line and center your text vertically